test_util/compareECL.cpp

```diff
@@ -70,14 +71,11 @@ static bool has_result_files(const std::string& rootName)
 {
     std::vector<std::string> extList = { "EGRID", "INIT", "UNRST", "SMSPEC", "RFT" };
 
-    for (const auto& ext : extList) {
-        std::ifstream is(rootName + '.' + ext);
-        if (is) {
-            return true;
-        }
-    }
-
-    return false;
+    return std::any_of(extList.begin(), extList.end(),
+                       [&rootName](const auto& ext)
+                       {
+                           return std::filesystem::exists(rootName + '.' + ext);
+                       });
```

Comment:
yes i am aware of the difference but it is a conscious change. I'd argue it actually does make a difference in this context. The intent is to check if files exists here. It could actually be kind of confusing if files are excluded because we cannot read them, as the message conveyed to the user would be 'No files found for reference case' rather than 'Failed to open file xxx' which will be conveyed when opened through the proper mechanisms.